community/doc_guidelines/general.md

```diff
@@ -0,0 +1,97 @@
+# General Guidelines for Splinter Documentation
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+The Splinter documentation guidelines are a work in progress.
+For items not covered here, see
+[plainlanguage.gov](https://www.plainlanguage.gov/guidelines/),
+the [Merriam-Webster's online dictionary](https://www.merriam-webster.com/),
+and the [Google developer documentation style
+guide](https://developers.google.com/style).
+Many thanks to the Google writers who provide this clear, comprehensive guide.
```

Comment:
"this clear, comprehensive guide" sounds like you're referring to this guide (the one you're writing). Might want to reword slightly. :)

Reply:
Didn't realize most of the content on this page is from that guide; seems fine on second read.